### DISCUSSION TITLE
Help with generating METviewer scorecard workflow

### DISCUSSION THREAD

**Discussion by bmatilla** [dtcenter/METplus#2292] [Statistical Computation] [answered]
Hi all,

So as I am getting spun back up with a recently published version of METviewer, I have been struggling to get a scorecard created with some data that I was able to generate from running METplus. I see that METcalcpy is needed now for computing some of the statistics needed for the scorecard, but I can't find documentation on an example workflow to help get there. Would someone be able to help advise on how to go about creating the scorecard with this? The MET suite of software I am using is all under the METplus coordinated release v5.0. Thanks!

**Comment by JohnHalleyGotway** [comment, marked as answer]
@bmatilla I see you have a question about generating scorecards using METviewer. I contacted @TatianaBurek about this, as she's the lead on the METviewer repository. She confirms that the steps for creating a scorecard include:

1. Load MET output into a METdataio database.
2. Create an XML to define the rows and columns of the scorecard along with plotting options.
3. Run the `mv_scorecard.sh` script to pull data from the database, aggregate the data, and assess statistical significance of the differences.

All documentation and examples are here:
   https://metviewer.readthedocs.io/en/latest/Users_Guide/scorecard.html#example

**Comment by bmatilla** [comment]
Hi John and Tatiana,

Thanks for the quick response. For some reason I thought there were other steps involved with METcalcpy but hadn't realized it's already embedded within the scorecard logic. I was able to get the scorecard working on my end after a few modifications to a previous xml template I had.

**Reply by JohnHalleyGotway** [reply]
Fantastic! I'm excited to hear that the METviewer scorecard is being used there! Thanks for letting us know. I'll go ahead and mark this discussion as being answered.
